scripts/header_footer_git_push_changes.sh

```diff
@@ -6,11 +6,13 @@ REPOS="
 ../directory-ui-buyer
 ../directory-ui-export-readiness
 ../navigator"
-IFS= read -r -p "Enter commit message: " commitmsg
+IFS= read -r -p "	Enter commit message: " commitmsg
+IFS= read -r -p "	Enter JIRA ticket number (e.g. ED-0000): " ticketnum
 for dir in $REPOS; do
-	echo "Switching to repo $dir"
+	echo "	Switching to repo $dir"
 	cd $dir
 	git add -A
 	git commit -m "$commitmsg"
 	git push -u origin $(git rev-parse --abbrev-ref HEAD)
+	git pull-request -b master -m "$(printf "$(git rev-parse --abbrev-ref HEAD)\n\nhttps://uktrade.atlassian.net/browse/$ticketnum")"
```

Comment:
that seems to be creating a PR in bitbucket (I assume because Atlassian make bitbucket, and atlassian.net is in play here).